src/components/views/rooms/SendMessageComposer.tsx

```diff
@@ -351,13 +351,15 @@ export class SendMessageComposer extends React.Component<ISendMessageComposerPro
                 }
                 break;
             case KeyBindingAction.CancelReplyOrEdit:
-                dis.dispatch({
-                    action: "reply_to_event",
-                    event: null,
-                    context: this.context.timelineRenderingType,
-                });
-                event.preventDefault();
-                event.stopPropagation();
+                if (!!this.context.replyToEvent) {
```

Comment:
`RoomContext` doesn't appear to *have* a `replyToEvent`. Should this be `this.props.replyToEvent`?

Reply:
It does though, this is from the same file.

![image](https://github.com/matrix-org/matrix-react-sdk/assets/2403652/88d7a529-d4b6-4101-ae08-30f6d7f279cb)

Reply:
so what, the type definition is wrong?

Is there a reason to prefer the value from the context rather than the one in the `props`?

Reply:
The type definition is right... 

![image](https://github.com/matrix-org/matrix-react-sdk/assets/2403652/63b27ec0-53d5-4eb1-9461-c68b44b6970d)

It is the entirety of IRoomState + an optional `threadId`

![image](https://github.com/matrix-org/matrix-react-sdk/assets/2403652/b33d3477-94ff-44bd-96ab-cb62ec4eb44f)

Reply:
ohh, I can't read. Sorry.

Reply:
> Is there a reason to prefer the value from the context rather than the one in the props?

They should be the same, I was just matching the existing code I shared in a screenshot above. We could likely get rid of the woven through props value in favour of the context one